src/components/LoginTitleText/styles.module.css

```diff
@@ -1,3 +1,3 @@
 .text {
-  font-size: 64px;
-}
+  font-size: 2.5rem;
```

Comment:
Yo I never knew rem existed, I'll be using it myself 👍

Reply:
I just discovered it too! It's super useful